X10D/src/Collections/Int16Extensions.cs

```diff
@@ -17,9 +22,9 @@ public static class Int16Extensions
     [Pure]
     public static bool[] Unpack(this short value)
     {
-        Span<bool> buffer = stackalloc bool[Size];
-        value.Unpack(buffer);
-        return buffer.ToArray();
+        bool[] ret = new bool[Size];
```

Comment:
Since the assigned type here is apparent, `var` is preferred rather than explicit typing.